lib/common/format_text.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class FormatText extends StatelessWidget {
+  final String text;
+  final Colors textColor;
+  final double fontSize;
+  final FontWeight fontWeight;
+
+  FormatText({
+    Key key, 
+    @required this.text, 
+    this.textColor, 
+    this.fontSize, 
+    this.fontWeight
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      text, 
+      style: GoogleFonts.muli(
+        textStyle: TextStyle(
+          color: textColor ?? Colors.black,
+          letterSpacing: .5, 
+          fontSize: fontSize ?? 40.0,  
+          fontWeight: fontWeight ?? FontWeight.bold
```

Comment:
Instead of us repeating this style everywhere, I created this `FormatText` widget that can be used throughout the codebase to handle the styling of the text. 

The next step would be to create a config to standardize certain text props. For instance: 
```dart
const textStylesConfig = {
   HEADER: {
       fontColor: Colors.black, 
       fontSize: ...
       fontWeight: ...
   },
   SUBHEADER: {
       fontColor: Colors.grey, 
       fontSize: ...
       fontWeight: ...
   }
   ...
}
```

Reply:
Good catch - I'll add it in!

Reply:
God bless! It was so annoying having to style every time.

Reply:
God bless! It was so annoying having to style every time.

Reply:
![image](https://user-images.githubusercontent.com/23146829/87352023-9d2b4600-c517-11ea-9a20-999c74bb5886.png)